package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery-focuspoint",
+  "version": "1.1.3",
+  "description": "jQuery plugin for 'responsive cropping'. Dynamically crop images to fill available space without cutting out the image's subject. Great for full-screen images.",
+  "main": "js/jquery.focuspoint.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jonom/jquery-focuspoint.git"
+  },
+  "keywords": [
+    "\"crop\"",
+    "\"cropping\"",
+    "\"image\"",
+    "\"responsive\"",
+    "\"fill\"",
+    "\"background\"",
+    "\"full-screen\"",
+    "\"focus\"",
+    "\"focuspoint\""
+  ],
+  "author": "\"Jonathon Menz (http://jonathonmenz.com)\"",
```

Comment:
I linked to the author spec in the last review, please check it out: https://docs.npmjs.com/files/package.json#people-fields-author-contributors `author` is an object with `name` and `url` properties. So this should be
```json
"author": {
	"name": "Jonathon Menz",
	"url":  "http://jonathonmenz.com"
}
```